libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss

```diff
@@ -0,0 +1,93 @@
+.container {
+  top: 0px;
+  left: 0px;
+  width: 100vw;
+  height: 100vh;
+  background: #DBC2F5 0% 0% no-repeat padding-box;
```

Comment:
For reuse of colors, use CSS variables specified in elements/theming/_variables.scss for specifying colors/background colors.